code/game/jobs/job/engineering.dm

```diff
@@ -32,6 +32,7 @@
 			if(4) H.equip_or_collect(new /obj/item/weapon/storage/backpack/satchel(H), slot_back)
 		H.equip_or_collect(new /obj/item/clothing/under/rank/chief_engineer(H), slot_w_uniform)
 		//H.equip_or_collect(new /obj/item/device/pda/heads/ce(H), slot_l_store)
+		H.equip_or_collect(new /obj/item/device/t_scanner/adv(H), slot_r_store)
```

Comment:
Should probably spawn in the CE locker instead, like the advanced device analyzer and advanced tool-belt.